src/main/java/cubicchunks/util/MathUtil.java

```diff
@@ -112,6 +112,26 @@ public static double gaussianProbabilityDensity(double x, double mean, double st
         return exp(-(x - mean) * (x - mean) / (2 * stdDev * stdDev)) /
                 (sqrt(2 * Math.PI) * stdDev);
     }
+	
+    public static double gaussianProbabilityCyclic(int x, int mean, double stdDev, int spacing) {
```

Comment:
First this is not true gaussian probability density. It should be at least explained in javadoc

And I think a  much simpler approach that wouldn't take such long comment to explain would be to use the existing gaussian probability function, and give it `(x + spacing/2) mod spacing)` as `x` with mod being one from util method that makes modulo always positive (because `x % y` is nagative for negative `x`)